lib/kitchen/verifier/pester.rb

```diff
@@ -151,145 +204,156 @@ def run_command_script
         CMD
       end
 
+      def get_powershell_modules_from_nugetapi
+        # don't return anything is the modules subkey or bootstrap is null
+        return if config.dig(:bootstrap, :modules).nil?
+
+        bootstrap = config[:bootstrap]
+        # if the repository url is set, use that as parameter to Install-ModuleFromNuget. Default is the PSGallery url
+        repository_url = bootstrap[:repository_url]
+        gallery_url_param = if repository_url
+          "-GalleryUrl '#{repository_url}'"
+        else
+          ""
+        end
```

Comment:
It would break the powershell, but similarly as typing the wrong URL...
The ruby side should be fine.
Not sure what would be better.... %{} maybe...?

Reply:
Perhaps build it into a hash for splatting? 🤔

Reply:
feels overkill and less visible if we create a ruby hash before converting it to PS then splat.
Maybe even more fragile...

Reply:
I haven't changed this for now, as I don't see what would be really better...